packages/flutter/test/widget/draggable_test.dart

```diff
@@ -944,6 +944,65 @@ void main() {
     await firstGesture.moveBy(new Offset(100.0, 0.0));
     await secondGesture.up();
   });
+
+  testWidgets('DragTarget does not set state when remove from the tree', (WidgetTester tester) async {
+    List<String> events = <String>[];
```

Comment:
Please add a comment about this being a regression test for https://github.com/flutter/flutter/issues/6128